examples/todo-list/flutter-todo-list/lib/functions/auth.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:supabase_flutter/supabase_flutter.dart';
+
+class AuthSupabase {
+  static final client = SupabaseClient(
+      "https://newknrbzotfpvatooqxz.supabase.co",
+      "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6Im5ld2tucmJ6b3RmcHZhdG9vcXh6Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2NTQwMjM2ODYsImV4cCI6MTk2OTU5OTY4Nn0.KI6TzzzailhNRfRAeJMIDtMVZSBZbaliDK2K_J0VVn8");
+  static Future<bool> createAccount(
+      {required String email, required String password}) async {
+    final res = await client.auth.signUp(email, password);
+    final user = res.data?.user;
+    final error = res.error;
+    print(user);
+    print(error);
+    if (error == null) {
+      return true;
+    } else {
+      return false;
+    }
+  }
+
+  static Future<String?> loginUser(
+      {required String email, required String password}) async {
+    final res = await client.auth.signIn(email: email, password: password);
+    final user = res.data?.user;
+    final error = res.error;
+    return user?.id;
+  }
+
+  static logOut(BuildContext context) async {
+    await client.auth.signOut();
+    Navigator.pushNamed(context, '/');
```

Comment:
If we `push` a route after logging out, the user can go back to the previous page. It would be better if we make sure the user cannot go back to the previous page after logging out. 

Also, personally not a fan of putting router related logic into Auth functions. Maybe it's better to handle the router related stuff at where `logOut()` is being called!

Reply:
DONE ✅